### ISSUE TITLE
Incorrect latency handling

### ISSUE THREAD

**Issue by hfujita** [hfujita/pulseaudio-raop2#4]
Currently there is a large latency between playback start from a software and actual start from a device. I believe most of the latency is due to several handshaking required by the protocol and thus not reducible. 
The issue is that software is not (accurately) aware of this large latency, so for example movie players there will be a gap between picture and sound.

**Comment by singpolyma** [comment]
This is the case even with official Apple clients.

**Comment by hfujita** [comment]
@singpolyma I don't agree. For instance the YouTube player on my iPad does the perfect lip-sync.

**Comment by bendlas** [comment]
Just crosslinking the ml thread: http://lists.freedesktop.org/archives/pulseaudio-discuss/2013-October/018951.html

@hfujita did you have any progress on this? Being able to listen to music is nice, but watching video would be even better. Do you have a technique to at least set the delay manually?

**Comment by hfujita** [comment]
@bendlas If you are okay with manual adjustment you can tweak the following number.
https://github.com/hfujita/pulseaudio-raop2/blob/hf/raop2-v2-v5.99.2/src/modules/raop/raop-client.c#L545

88200 means 44100(sampling rate)*2, which is 2 seconds.
You can reduce the delay by tweaking this value, but for buffering reasons I don't think this could be zero.

One way to enjoy video is to use a video player which allows you to adjust vide/audio delay. I believe VLC has this kind of option.
